toolkits/springer/packages/springer-header/scss/50-components/_header.scss

```diff
@@ -3,9 +3,9 @@
 }
 
 .c-header {
-	background: $header--background;
+	background-color: #fff;
```

Comment:
Why not change the value in the settings?

Reply:
Oh yeah derrrr, for some reason that didn't even occur